src/local_sensitivity/sensitivity_interface.jl

```diff
@@ -27,7 +27,7 @@ function _adjoint_sensitivities(sol,sensealg,alg,g,t=nothing,dg=nothing;
   tstops = ischeckpointing(sensealg, sol) ? checkpoints : similar(sol.t, 0)
   adj_sol = solve(adj_prob,alg;
                   save_everystep=false,save_start=false,saveat=eltype(sol[1])[],
-                  abstol=abstol,reltol=reltol,kwargs...)
+                  tstops=tstops,abstol=abstol,reltol=reltol,kwargs...)

```

Comment:
The `tstops` here seems to be necessary to make checkpointing work as expected. Otherwise, these tests:
https://github.com/SciML/DiffEqSensitivity.jl/blob/e0ceda43ea1184ec0a6c3eab202f7f5099401854/test/local_sensitivity/adjoint.jl#L610
are failing.